services/construction_service.go

```diff
@@ -71,7 +71,7 @@ func (s *ConstructionAPIService) ConstructionMetadata(ctx context.Context, reque
 		return nil, ErrUnavailableOffline
 	}
 
-	utxoView, err := s.node.GetMempool().GetAugmentedUniversalView()
+	utxoView, err := lib.NewUtxoView(s.node.GetBlockchain().DB(), s.node.Params, s.node.GetBitcoinManager())
```

Comment:
Sooo for construction don't you still wanna use the mempool? If you don't then calling this 2 times will result in them both trying to spend the same UTXO.